products/fc_ls.odc_product.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: fc_ls
+description: Landsat Fractional Cover
```

Comment:
Is this consistent with the Landsat WOfS description?

Reply:
WOfS is: "Historic Flood Mapping Water Observations from Space"